### ISSUE TITLE
Vaccination Nation isn't updating properly

### ISSUE THREAD

**Issue by tolomaps** [geog575-spring2016/geog575-spring2016.github.io#2]
@amandelman I can't figure out why it's not working when it's merged. The latest version worked when I ran it locally, but if you look at pull request #1, it looks like the only file that was changed was the .swf file. 
<img width="914" alt="screen shot 2016-05-18 at 12 17 45 pm" src="https://cloud.githubusercontent.com/assets/1679587/15368281/9d1c154e-1cf2-11e6-8af7-6faf1305d3f8.png">

Maybe try making a small change to the latest version of the code, like adding a comment, and then create another PR?

**Comment by tolomaps** [comment]
For the record, here's the current web address for the project: http://geog575-spring2016.github.io/vaccination-nation/

**Comment by amandelman** [comment]
Yeah—I noticed that and decided to make the PR anyway and see what would
happen. I'll futz.

A

## 

Adam Mandelman
Managing Editor, Wisconsin 101 http://www.wi101.org/
www.adammandelman.net
Department of Geography
University of Wisconsin - Madison
mandelman@wisc.edu

**Comment by tolomaps** [comment]
Success! I ended up just creating a new folder with your project's new title and putting it there: http://geog575-spring2016.github.io/immunity-mutiny/

**Comment by tolomaps** [comment]
^ @amandelman
